### ISSUE TITLE
cannot annotate pdf anymore, login requested

### ISSUE THREAD

**Issue by avillani** [elias-sundqvist/obsidian-annotator#179]
Hello,

A few days ago, the plugin was working perfectly. But I discovered yesterday that I could not annotate anymore, and that my previous annotations were not showing up in the side bar when I was in annotation mode. (The annotations are present in the markdown file, that's not the problem)

- I have an error that shows up when trying to highlight something "Please log in to create annotations or highlights."
- When opening console, I see more error messages:
![image](https://user-images.githubusercontent.com/23527241/173017037-3ecd2809-fc0e-4ab6-b60a-dd743ce40108.png)
- and warnings
![image](https://user-images.githubusercontent.com/23527241/173017597-9e266d0f-3e14-4047-b3f7-52eff0faca5e.png)

It seems that something changed on the API side of hypothes.is ?

Thanks !

**Comment by avillani** [comment]
hi again,

I think I solved it by pure luck, it may not have come from Hypothesis.

While I was considering switching to Zotero (instead of JabRef), I played with the obsidian plugin [obsidian-zotero-integration](https://github.com/mgmeyers/obsidian-zotero-integration)
Ii was failing to load on my system, and in an issue, they  recommended to change the installer version, which seems to be different from the Obsidian version itself.
It indeed worked with the obsidian-zotero-integration, so I thought, let's give it another try for obsidian-annotator. And now it's working again !

It's the only thing I changed, so if anyone comes across this problem, maybe it will help.
